language/diem-tools/transaction-replay/Cargo.toml

```diff
@@ -25,7 +25,7 @@ move-vm-types = { path = "../../move-vm/types" }
 move-core-types = { path = "../../move-core/types" }
 move-vm-runtime = { path = "../../move-vm/runtime" }
 move-vm-test-utils = { path = "../../move-vm/test-utils" }
-resource-viewer = { path = "../../tools/resource-viewer" }
+diem-resource-viewer = { path = "../../tools/diem-resource-viewer" }
```

Comment:
As someone new to this project, just renaming crates like this is so so helpful to understand what belongs in which project 👍

Reply:
You might also find it useful to read `x.toml`, particularly the section named `[workspace.move-to-diem-deps]`. This will tell you which crates in the `language` directory are actually Diem! Note a lot of the tests are considered Diem right now due to legacy reasons, but as you know we do plan to rework them and will likely get the Move and Diem parts separated through the process.